### PR TITLE
Update can-event enter and radiochange

### PR DESCRIPTION
**Pull request by green3g**
..to match canjs/canjs versions

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -34,8 +34,8 @@
     "can-attribute-encoder": "^0.3.0",
     "can-compute": "^3.3.1",
     "can-event": "^3.5.0",
-    "can-event-dom-enter": "^1.0.2",
-    "can-event-dom-radiochange": "^1.0.2",
+    "can-event-dom-enter": "^1.0.4",
+    "can-event-dom-radiochange": "^1.0.4",
```
Comment (chasenlehara) [inline package.json:38]: Hi @roemhildtg, I would not expect it to be necessary for us to update these ranges… what does this fix?

**Comment by green3g** [comment]
This fixes the error: `Error: You can't have two versions of can-dom-data-state, check your dependencies` because the can-event-dom-enter depends on an older version of can-dom-data-state.

**Comment by phillipskevin** [comment]
Would removing the `package-lock.json` file allow these to be updated automatically? (since version `1.04` matches the semver range `^1.0.2` which was there previously)

**Comment by green3g** [comment]
Good point. I'll close this and the jquery one also.

**Pull request closed without merging**